csharp/src/Ice/InputStream.cs

```diff
@@ -2139,15 +2139,31 @@ public void CopyTo(T[] array, int arrayIndex)
 
         private sealed class Enumerator<T> : IEnumerator<T>
         {
-            public T Current { get; private set; }
+            private T _current;
```

Comment:
Should be listed with the other private fields.

Reply:
fixed